scripts/lib/install-node

```diff
@@ -57,6 +57,12 @@ if [ "$current_node_version" != "v$node_version" ] || ! [ -L "$node_wrapper_path
     ln -nsf "$NODE_BIN" /usr/local/bin/node
     ln -nsf "$(dirname "$NODE_BIN")/npm" /usr/local/bin/npm
     ln -nsf "$(dirname "$NODE_BIN")/npx" /usr/local/bin/npx
+
+    # Tell NVM that we don't want it messing around with $PATH, which
+    # can get us into trouble, if we've just upgraded but our parent
+    # env still has the old version first in its PATH.  Tell it to use
+    # the newly-symlinked one that it will find in /usr/local/bin
+    nvm alias default system
```

Comment:
We could do this before the `nvm install` command, so we don’t change `PATH` even if something goes wrong during installation.

Reply:
Yeah, this is probably a good idea.

Reply:
#18444.